src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -22,10 +22,19 @@ trait WorkspaceService extends HttpService with FireCloudDirectives {
   private final val ApiPrefix = "workspaces"
   private implicit val executionContext = actorRefFactory.dispatcher
 
-  val routes = createWorkspaceRoute
+  val routes = {optionsRoute ~ createWorkspaceRoute ~ listWorkspacesRoute}
 
   lazy val log = LoggerFactory.getLogger(getClass)
 
+  def optionsRoute: Route =
```

Comment:
Any reason to not swaggerize this (and the equivalent in MethodsService)? We may only use this for cors preflighting, but might as well be transparent that it exists.